tests/modules/sharedaddy/test_class.recaptcha.php

```diff
@@ -0,0 +1,139 @@
+<?php
+require dirname( __FILE__ ) . '/../../../modules/sharedaddy/recaptcha.php';
+
+class WP_Test_Jetpack_ReCaptcha extends WP_UnitTestCase {
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->site_key   = defined( 'RECAPTCHA_PUBLIC_KEY' ) ? RECAPTCHA_PUBLIC_KEY : 'sitekey';
+		$this->secret_key = defined( 'RECAPTCHA_PRIVATE_KEY' ) ? RECAPTCHA_PUBLIC_KEY : 'secretkey';
```

Comment:
Done in https://github.com/gedex/jetpack/commit/a197f2d8b14b271cf7efe9ac2709fde6a4ed34d9.